src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/tests/Sidebar.test.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import {render} from 'enzyme';
+import Sidebar from '../Sidebar';
+import sidebarStore from '../stores/SidebarStore';
+import sidebarViewRegistry from '../registries/SidebarViewRegistry';
+
+const component = (props) => (<h1>{props.title}</h1>);
+
+jest.mock('../stores/SidebarStore', () => ({}));
+
+jest.mock('../registries/SidebarViewRegistry', () => ({
+    get: jest.fn(),
+}));
+
+test('Render correct sidebar view', () => {
+    sidebarStore.view = 'preview';
+    sidebarViewRegistry.get.mockReturnValue(component);
+
+    const view = render(<Sidebar />);
+    expect(view).toMatchSnapshot();
```

Comment:
Usually we don't introduce another variable just for the return value of `render`. Sorry for not seeing that earlier.